### ISSUE TITLE
right click export or copy not working

### ISSUE THREAD

**Issue by rachamaduguravi** [finos/perspective#2139]
### Steps to Reproduce:
 1. extract superstore.zip and make available the 2 files via webserver
 2. load index.html
 3. right click and choose export
 4. right click and choose copy

### Expected Result:
... export needed to initiate

### Actual Result:
... console shows below error

```
Uncaught (in promise) TypeError: n.widget.viewer.unsafe_get_model is not a function
    at execute (commands.js:20:60)
    at vr.execute (index.ts:425:27)
    at Qi.triggerActiveItem (menu.ts:319:21)
    at Qi._evtMouseUp (menu.ts:703:10)
    at Qi.handleEvent (menu.ts:487:14)
execute @ commands.js:20
execute @ index.ts:425
triggerActiveItem @ menu.ts:319
_evtMouseUp @ menu.ts:703
handleEvent @ menu.ts:487
```

[superstore.zip](https://github.com/finos/perspective/files/10993424/superstore.zip)

**Comment by naveen-kumawat** [comment]
The error message "Uncaught (in promise) TypeError: n.widget.viewer.unsafe_get_model is not a function" suggests that the function unsafe_get_model is not defined in the object n.widget.viewer. This error could occur due to a number of reasons, such as:

1. The function unsafe_get_model is not defined or is not accessible from within the current scope. You may want to verify that the function exists and is being imported or included correctly in your code.
2. The object n.widget.viewer is not properly defined or instantiated. Ensure that the object exists and is initialized before calling the unsafe_get_model function.
3. The data type of the object n.widget.viewer is incorrect. Ensure that the object is of the expected data type, and that it has the necessary properties and methods.
4. There may be a compatibility issue between the version of the code and the dependencies it relies on. Try updating your dependencies or rolling back to a previous version of the code to see if this resolves the issue.

Without more information about your specific use case and code, it's difficult to provide a more specific solution. However, you may want to try debugging the code by checking the console logs, adding breakpoints, or stepping through the code to identify the root cause of the error.

**Comment by texodus** [comment]
This is fixed in #2214 and will be released in `2.1.0`
